presentation/src/main/java/com/knocklock/presentation/home/HomeViewModel.kt

```diff
@@ -36,29 +39,50 @@ class HomeViewModel @Inject constructor(
         .stateIn(
             scope = viewModelScope,
             started = SharingStarted.WhileSubscribed(5_000),
-            initialValue = LockScreen()
+            initialValue = LockScreen(),
         )
 
     val homeScreenUiState = combine(
         homeMenuList,
         tmpHomeScreen,
-        savedHomeScreenState
+        savedHomeScreenState,
     ) { menuList, tmpScreen, savedScreen ->
         HomeScreenUiState.Success(
             menuList = menuList.toImmutableList(),
-            lockScreen = if (tmpScreen is TmpScreenState.Custom) tmpScreen.screen else savedScreen
+            lockScreen = if (tmpScreen is TmpScreenState.Custom) tmpScreen.screen else savedScreen,
         )
     }.stateIn(
         scope = viewModelScope,
         started = SharingStarted.WhileSubscribed(5_000),
-        initialValue = HomeScreenUiState.Loading
+        initialValue = HomeScreenUiState.Loading,
     )
 
-    fun saveLockScreen() {
+    fun saveLockScreen(context: Context) {
         viewModelScope.launch {
             val value = tmpHomeScreen.value
             if (value is TmpScreenState.Custom) {
-                saveLockScreenCase(value.screen)
+                val uri = Uri.parse((value.screen.background as LockScreenBackground.LocalImage).imageUri)
+                if (uri.path == null) {
+                    return@launch
```

Comment:
```suggestion
                if (uri?.path == null) {
                    return@launch
```

현재 코드에서 uri이 플랫폼타입이어서 NullCheck 해주는 것이 안전해요

Reply:
java꺼라서 플랫폼타입이기 때문에 kotlin으로 변환해서 null check